recipes/automake/all/conanfile.py

```diff
@@ -90,34 +90,34 @@ def package_info(self):
         self.output.info("Appending PATH env var with : {}".format(bin_path))
         self.env_info.PATH.append(bin_path)
 
-        bin_ext = ".exe" if self.settings.os_build == "Windows" else ""
+        bin_ext = ".exe" if self.settings.os == "Windows" else ""
 
         aclocal = os.path.join(self.package_folder, "bin", "aclocal" + bin_ext)
-        if self.settings.os_build == "Windows":
+        if tools.os_info.is_windows:
```

Comment:
`tools.unix_path()` will have no effect if the OS is different from windows in Conan 1.25. Maybe we can wait a bit a remove this checks

Reply:
I waited 24 hours. Is that enough?

Reply:
there is a minor 1.25.1 waiting, so... but the issue with `unix_paths()` only affect in special case so I guess we can move forward this one